.p4a

```diff
@@ -0,0 +1,21 @@
+--window
+--bootstrap "webview"
+--package "org.learningequality.Kolibri"
+--name "Kolibri"
+--dist_name "kolibri"
+--private "src"
+--orientation sensor
+--requirements python3,android,pyjnius,genericndkbuild,sqlite3,cryptography,twisted,attrs,bcrypt,service_identity,pyasn1,pyasn1_modules,pyopenssl,openssl,six
+--android-api 30
+--minsdk 21
+--permission WRITE_EXTERNAL_STORAGE
+--permission ACCESS_NETWORK_STATE
+--permission FOREGROUND_SERVICE
+--service server:server.py
+--service remoteshell:remoteshell.py
+--presplash assets/launch-image.png
+--icon assets/icon.png
+--fileprovider-paths src/fileprovider_paths.xml
+--add-asset assets/_load.html:_load.html
```

Comment:
when compiling, no command found with  --add-asset assets/_load.html:_load.html

Reply:
Make sure your dependencies are properly set - this is a valid argument for Python for Android.